lib/Backend/Lower.cpp

```diff
@@ -9041,8 +9039,20 @@ Lowerer::LowerStElemI(IR::Instr * instr, Js::PropertyOperationFlags flags, bool
         helperMethod == IR::HelperOp_InitClassMemberSetComputedName
         );
 
+    Assert(dst->IsIndirOpnd());
```

Comment:
nit: no need to add the assert because `AsIndirOpnd` already has that same assert